### DISCUSSION TITLE
LTAR API [May 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5761] [Discord Threads]
> 🙋‍♂️ **kerrygotten**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Apologies for the confusion. childId & parentId are table ID's  
#  
> 🙋‍♂️ **kerrygotten**  
#### Thank you that fixed it.  
#  
> 🙋‍♂️ **kerrygotten**  
#### However doing this the 'column_name' field does not work, it doesn't do anything. the column_name is null, is that normal? If I set it as something different from the title the title shows up instead of column_name  
#  
> 🙋‍♂️ **kerrygotten**  
#### Trying to update the column also gives me a 400.   
#### `   
####     api.dbTableColumn.create(tb.id, {   
####       column_name: 'Sheet 1',   
####       childId: sheet1.foreignKey,   
####       parentId: sheet2.foreignKey,   
####       title: 'sheet1',   
####       uidt: 'LinkToAnotherRecord',   
####       type: 'mm'   
####     }).then(res => {   
####    
####     api.dbTableColumn.update(columnId, {   
####       column_name: 'Sheet 1',   
####       childId: childTableId,   
####       parentId: parentTableId,   
####       title: 'sheet1',   
####       type: "mm",   
####       uidt: "LinkToAnotherRecord",   
####     }).catch(err => {   
####       console.log(err)   
####     })   
#### })   
#### `  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### `Title` is UI display name; `column_name` represents name as it should appear in database.  
#  
> 🙋‍♂️ **kerrygotten**  
#### Thanks yeah I'm a dumbass haha  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1098422049468325898)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
